python/ray/_raylet.pyx

```diff
@@ -4429,13 +4436,15 @@ cdef class CoreWorker:
                     name, ray_namespace))
         check_status(named_actor_handle_pair.second)
 
-        return self.make_actor_handle(named_actor_handle_pair.first)
+        return self.make_actor_handle(named_actor_handle_pair.first,
+                                      weak_ref=True)
 
     def get_actor_handle(self, ActorID actor_id):
```

Comment:
Is this to handle `runtime_context().current_actor`?

Reply:
This is for `get_actor` by name.